nextmv/nextmv/default_app/requirements.txt

```diff
@@ -0,0 +1,2 @@
+nextmv>=0.29.3
+plotly>=6.2.0
```

Comment:
Feels like a bit of a heavy dependency to bring in for the init app. Getting this:

```bash
📦 Packaged application (18.06 MiB, 4184 files).
```

But I don't have a strong opinion on it. It does show the use of visuals (one of them). geojson would be cheaper dependency wise, but :shrug: